lib/IRGen/GenStruct.cpp

```diff
@@ -339,6 +340,50 @@ namespace {
     }
   };
 
+  class AddressOnlyClangRecordTypeInfo final
+      : public StructTypeInfoBase<AddressOnlyClangRecordTypeInfo, FixedTypeInfo,
+                                  ClangFieldInfo> {
+    const clang::RecordDecl *ClangDecl;
+
+  public:
+    AddressOnlyClangRecordTypeInfo(ArrayRef<ClangFieldInfo> fields,
+                                   llvm::Type *storageType, Size size,
+                                   Alignment align,
+                                   const clang::RecordDecl *clangDecl)
+        : StructTypeInfoBase(StructTypeInfoKind::AddressOnlyClangRecordTypeInfo,
+                             fields, storageType, size,
+                             // Create an empty SpareBitVector with a width of
```

Comment:
This comment repeats what the code does -- everyone should be able to figure that out. Instead, could you add a comment that explains the reason for it?

Reply:
👍